### ISSUE TITLE
Object Explorer doesn't like scalar members of collections

### ISSUE THREAD

**Issue by TamarEGranor** [VFPX/ObjectExplorer#40]
## ℹ Computer information

- VFP version: 9 SP2 Build 7423
- ObjectExplorer version: 3.4

## 📝 Provide detailed reproduction steps (if any)

1. Create a collection whose members are scalars, not other object. For example, run this code in the Command Window:

```
oColl = CREATEOBJECT("collection")
oColl.Add("Lucy")
oColl.Add("Ricky")
oColl.Add("Ethel")
oColl.Add("Fred")
```

2. Open Object Explorer passing this collection as a parameter.
3. Click on any of the scalar items.

### ✔️ Expected result

Right-pane would show the scalar value (as in Object Inspector)

### ❌ Actual result

Error message: 

Program Error
Cancel
Suspend
Ignore
Help
Member TAG does not evaluate to an object.

## 📷 Screenshots

_Are there any useful screenshots? WinKey+Shift+S and then just paste them directly into the form_

## Notes
Obviously this is a trivial case. But it's not out of the question to have a complex object hierarchy in which a collection contains scalars rather than objects.

**Comment by Jimrnelson** [comment]
Ooops!

Trivial cases that demonstrate a real problem are the easiest to solve.

Not much commentary on this project, glad it’s in use.

**Comment by Jimrnelson** [comment]
Fixed in today's release 3.4.02
